Sources/PBXAggregateTarget.swift

```diff
@@ -11,32 +11,6 @@ public class PBXAggregateTarget: PBXTarget {
 extension PBXAggregateTarget: PlistSerializable {
     
     func plistKeyAndValue(proj: PBXProj) -> (key: CommentedString, value: PlistValue) {
-        var dictionary: [CommentedString: PlistValue] = [:]
-        dictionary["isa"] = .string(CommentedString(PBXAggregateTarget.isa))
-        let buildConfigurationListComment = "Build configuration list for PBXAggregateTarget \"\(name)\""
-        dictionary["buildConfigurationList"] = .string(CommentedString(PBXNativeTarget.isa,
```

Comment:
@pepibumur Let me know if this `PBXNativeTarget.isa` was there for a reason. Looks like it was a bug, and is supposed to be `buildConfigurationList`

Reply:
Yeah, it's definitively a bug. Good catch.